tex/pgfcircpath.tex

```diff
@@ -449,16 +453,17 @@
 
 \def\pgfcircdiodestylemacro{\ifpgf@circuit@strokediode stroke \else\ifpgf@circuit@fulldiode full \else empty \fi\fi}
 
+\compattikzset{diode/.style = {\comnpatname \pgfcircdiodestylemacro diode}}
 \compattikzset{Schottky diode/.style = {\comnpatname \pgfcircdiodestylemacro Schottky diode}}
 \compattikzset{Zener diode/.style = {\comnpatname \pgfcircdiodestylemacro Zener diode}}
 \compattikzset{ZZener diode/.style = {\comnpatname \pgfcircdiodestylemacro ZZener diode}}
 \compattikzset{tunnel diode/.style = {\comnpatname \pgfcircdiodestylemacro tunnel diode}}
 \compattikzset{photodiode/.style = {\comnpatname \pgfcircdiodestylemacro photodiode}}
 \compattikzset{led/.style = {\comnpatname \pgfcircdiodestylemacro led}}
 \compattikzset{varcap/.style = {\comnpatname \pgfcircdiodestylemacro varcap}}
-\compattikzset{diode/.style = {\comnpatname \pgfcircdiodestylemacro diode}}
```

Comment:
Why are you removing `diode` here? I think that this is the reason why the build fails...